src/luawasm.ts

```diff
@@ -280,6 +283,7 @@ export default class LuaWasm {
         this.lua_pushnumber = this.cwrap('lua_pushnumber', null, ['number', 'number'])
         this.lua_pushinteger = this.cwrap('lua_pushinteger', null, ['number', 'number'])
         this.lua_pushlstring = this.cwrap('lua_pushlstring', 'string', ['number', 'string|number', 'number'])
+        this.lua_ptr_pushlstring = this.cwrap('lua_pushlstring', 'string', ['number', 'string|number', 'number'])
```

Comment:
The return type here is string, but you added the return as number on the field?